language/diem-framework/modules/0L/Wallet.move

```diff
@@ -285,6 +303,7 @@ module Wallet {
       return pending
     }
     
+    // Todo: Where is the "maybe" here?
```

Comment:
@0o-de-lally FYI.

Reply:
good catch. I think that's historic. You can rename.

Reply:
Done.